plugins/MultiSites/angularjs/dashboard/dashboard-model.service.js

```diff
@@ -66,6 +66,57 @@
                 site.visits_evolution    = parseInt(site.visits_evolution, 10);
                 site.pageviews_evolution = parseInt(site.pageviews_evolution, 10);
                 site.revenue_evolution   = parseInt(site.revenue_evolution, 10);
+
+                if (site.hasOwnProperty('ratio') && site.ratio != 1) {
+                    var percent = NumberFormatter.formatPercent(Math.round((site.ratio * 100)));
+                    var metricName = null;
+                    var previousTotal = 0;
+                    var currentTotal = 0;
+                    var evolution = 0;
+                    var previousTotalAdjusted = 0;
+                    if (model.sortColumn == 'nb_visits' || model.sortColumn == 'visits_evolution') {
+                        previousTotal = NumberFormatter.formatNumber(site.previous_nb_visits);
+                        currentTotal = NumberFormatter.formatNumber(site.nb_visits);
+                        evolution = NumberFormatter.formatPercent(site.visits_evolution);
+                        metricName = _pk_translate("General_ColumnNbVisits");
+                        previousTotalAdjusted = NumberFormatter.formatNumber(Math.round(site.previous_nb_visits * site.ratio));
+                    }
+                    if (model.sortColumn == 'pageviews_evolution') {
+                        previousTotal = site.previous_Actions_nb_pageviews;
+                        currentTotal = site.nb_pageviews;
+                        evolution = NumberFormatter.formatPercent(site.pageviews_evolution);
+                        metricName = _pk_translate("General_ColumnPageviews");
+                        previousTotalAdjusted = NumberFormatter.formatNumber(Math.round(site.previous_Actions_nb_pageviews * site.ratio));
+                    }
+                    if (model.sortColumn == 'revenue_evolution') {
+                        previousTotal = NumberFormatter.formatNumber(site.previous_Goal_revenue);
```

Comment:
for the revenue values we could actually use `formatCurrency`, so it automatically adds the currency.

Reply:
Done, the evolution metric now passes the currency symbol as row metadata since each row is for a different site and might have different currency.